README.md

```diff
@@ -1,4 +1,4 @@
-Takeaway Challenge
+Makers End of Unit Challenge - Takeaway Challenge
 ==================
 ```
```

Comment:
Readme is nice and detailed. Love how you included the irb tests. This has definitely inspired me, hoping I can write such in-depth readme files in the future :)